cmd/diff.go

```diff
@@ -56,8 +71,11 @@ that will be created or updated or deleted.
 func init() {
 	rootCmd.AddCommand(diffCmd)
 	diffCmd.Flags().StringVarP(&diffCmdKongStateFile,
-		"state", "s", "kong.yaml", "file containing Kong's configuration.")
+		"state", "s", "kong.yaml", "file containing Kong's configuration")
 	diffCmd.Flags().BoolVar(&dumpConfig.SkipConsumers, "skip-consumers",
 		false, "do not diff consumers or "+
 			"any plugins associated with consumers")
+	diffCmd.Flags().BoolVarP(&diffCmdKongStateFileFromStdin, "read-state-from-stdin",
+		"-", false, "read Kong's configuration state from stdin (overrides "+
+			"--state flag)")
```

Comment:
Question: have you considered using `-` as an input to the -s/--state flag to read input from os.StdIn?
It would be nice if we can avoid this flag.

Reply:
The way it is now, if you pass `deck diff -s -`  it will read from `stdin` like in issue #10 . 

I added the other `BoolVal` flag just so you could call it with `deck diff --` or `deck diff --read-state-from-stdin` and still explicitly read from stdin.

Reply:
I feel that will confuse users and is unnecessary.
Could you please update the PR to remove this flag?
Also, please don't remove the `kong.yaml` file present in the repository.